config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
Why do we have to move the boot.img to a different place when it is already in the kernel snap? 

Also I am wondering if we can't determine the kernel + core snap to boot differently from the snapd database instead of repeating these information in a different place. This is different to how we do it for uboot and grub where we directly write these information into the bootloader configuration where we need a second step here to do this.